README.rst

```diff
@@ -69,6 +84,69 @@ Services:
 
 * Memcached
 
+Language Packages:
+
+* Frontend:
+
+  - ``npm clean-install`` (production)
+  - ``npm clean-install --dev`` (development)
+
+* Backend build:
+
+  - ``pip install -r requirements/edx/assets.txt``
+
+* Backend application:
+
+  - ``pip install -r requirements/edx/base.txt`` (production)
+  - ``pip install -r requirements/edx/dev.txt`` (development)
+
+Build Steps
+-----------
+
+Build static assets (for more details, see `building static
+assets`_)::
+
+  npm run build  # or, 'build-dev'
+
+Download locales and collect static assets (can be skipped for development
+sites)::
+
+  make pull_translations
+  ./manage.py lms collectstatic
+  ./manage.py cms collectstatic
+
+Provision your database::
+
+  # (TODO: we need a step here to create the database itself)
```

Comment:
```suggestion
  # (TODO: we need a step here to create the database itself)
```
Gotta look into what Tutor does and update this. @feanil , if you have the command you ran written down, let me know and I'll drop it in here.

Reply:
I've been running a minimal `docker compose` with the following config:

```
services:
  mysql:
    image: mysql:8.0
    environment:
      MYSQL_DATABASE: "edxapp"
      MYSQL_USER: "edxapp001"
      MYSQL_PASSWORD: "password"
      MYSQL_RANDOM_ROOT_PASSWORD: true
    ports:
      - 3306:3306
  mongo:
    image: mongo:7
    environment:
      MONGO_INITDB_DATABASE: 'edxapp'
    ports:
      - 27017:27017
    volumes:
      - type: bind
        source: mongo-entrypoints.d
        target: /docker-entrypoint-initdb.d/
        read_only: true
  memcache:
    image: memcached
    ports:
      - 11211:11211
```
with a mongo entrypoint file in the folder with the following content:
```
db.createUser(
  {
    user: "edxapp",
    pwd:  "password",   // cleartext dev password, change if you're gonna use this for production.
    roles: [
	     { role: "readWrite", db: "edxapp" },
           ]
  }
)
```